src/test/resources/item.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Siden dette er ekte data,  er det noe av dette som ikke burde ligget åpent på github ?

Reply:
Det er testdata fra den åpne sandboxen til Alma. Audun genererer fra ekte data for noe, men skal anonymisere der det trengs.

Reply:
altså guest sandboxen, skal ikke være noen kobling mot våre data så vidt jeg vet